src/components/helpers/Alert.js

```diff
@@ -98,7 +99,7 @@ function MySnackbar(props) {
             <CloseIcon className={classes.icon} />
           </IconButton>,
         ]}
-        {...other}
+        // {...other}
```

Comment:
What does this statement meant? If it was a mistake by previous developer, and it works correct, you can remove that(and also remove comment on 57 line)

Reply:
Was not sure what the previous developers wanted to do. Without this statement it works good. Already delete it.